views/index.handlebars

```diff
@@ -12,6 +12,11 @@
                 <td colspan="5">
                     <h1 style="color: #4A73E8;">Lighthouse Report - {{date}}</h1>
                     <p style="margin-top: -5px; font-size: 1em; padding-bottom: 20px; color: #62646A;">Please note that the <b>Performance</b> score should be considered an approximation and that 2-3 point swings in either direction is completely normal.</p>
+                    {{#if list}}
+                        {{#each list}}
+                            <a colspan="5" style="margin-bottom: -5px;padding: 20px; font-size: 1.5em; color: #404145;" href={{url}}>{{text}}</a>
```

Comment:
Easily support text that's not a link
```hbs
{{#each list}}
  <p>
  {{#if url}}
    <a style="margin-bottom: -5px; padding: 20px; font-size: 1.5em; color: #404145;" href={{url}}>{{text}}</a>
  {{else}}
    <span style="margin-bottom: -5px; padding: 20px; font-size: 1.5em;">{{text}}</span>
  {{/if}}
  </p>
{{/each}}
```
- Don't need the colspan attribute on anchor tag
- I think wrap each one in a paragraph to give each one a line, either than or a space with `&nbsp;` character